tsl/test/sql/CMakeLists.txt

```diff
@@ -134,6 +134,7 @@ set(SOLO_TESTS
     cagg_invalidation
     move
     reorder
+    transparent_decompression-${PG_VERSION_MAJOR}
```

Comment:
Why do we need this in SOLO_TESTS?

Reply:
Just a debugging leftover, I'll remove it